flask_caching/__init__.py

```diff
@@ -390,7 +413,7 @@ def decorated_function(*args, **kwargs):
                             args, kwargs, use_request=True
                         )
 
-                    if callable(forced_update) and forced_update() is True:
+                    if callable(forced_update) and (forced_update(*args, **kwargs) if wants_args(forced_update) else forced_update()) is True:
```

Comment:
The line is a bit to long. I'd like to keep the line length to 80.

I am using [black](https://github.com/python/black) to keep my python files nicely formatted (we gotta add this to our not yet existing CONTRIBUTING file :P)